app/sagas/login.js

```diff
@@ -240,6 +241,7 @@ const handleLoginSuccess = function* handleLoginSuccess({ user }) {
 			yield put(inviteLinksRequest(inviteLinkToken));
 		}
 		yield showSupportedVersionsWarning(server);
+		yield resendFailedMessages();
```

Comment:
Move to below `yield fork(fetchUsersRoles);` with a fork, so it doesn't `await` this saga.

Reply:
Sure